doc/releases.md

```diff
@@ -186,6 +186,14 @@ informed perspective, such as a member of the NAN team.
 see a need to bump `NODE_MODULE_VERSION` then you should consult the TSC.
 Commits may need to be reverted or a major version bump may need to happen.
 
+*Note*: The Node.js ecosystem is reliant on ABI compatibility within a Semver
```

Comment:
Micro-nit: Remove `*Note*:` here and in the paragraph above. Just say what you have to say.